frontend/src/components/ScheduledEventsTable/ScheduledEventsTable.tsx

```diff
@@ -1,15 +1,28 @@
-import React, { FunctionComponent, ReactNode } from 'react';
+import classNames from 'classnames';
+import React, { FunctionComponent, ReactNode, useState } from 'react';
 import { Table } from 'react-bootstrap';
+import { useHistory } from 'react-router-dom';
 import { ScheduledEvent } from '../../types/scheduledEvents';
 import { ScheduledEventsTableRow } from '../ScheduledEventsTableRow';
-
 export interface ScheduledEventTableProps {
   currentPage: number;
   pageLimit: number;
   events: Array<{}>;
 }
 export const ScheduledEventsTable: FunctionComponent<ScheduledEventTableProps> = (props) => {
   const offset = (props.currentPage - 1) * props.pageLimit;
+  const [activeRow, setActiveRow] = useState(0);
+  const history = useHistory();
+
+  const handleClick = (id: number) => (): void => {
+    if (id !== activeRow) {
+      setActiveRow(id);
+      history.push({
+        pathname: '/scheduledevents/',
+        state: { rowId: id },
+      });
```

Comment:
Ah, so that's what the history was being used for ... this won't do ... find another way. I don't know what shape the ScheduledEvents API returns them, but it should be providing an ID ... if it's not, please make sure it is. It's that ID that you need to capture on click.